components/scrollable-container/script.js

```diff
@@ -46,6 +46,42 @@ class ScrollableContainer extends BaseComponent {
         components.waitForFrames(() => this.scrollbar.scrollToPercents(this._scrollPos));
     }
 
+    /**
+     * Will scroll the container to a passed percents
+     * @param {string|number} value
+     * @returns {undefined}
+     */
+    scrollToPercents(value) {
+        value = parseFloat(value);
+        if (isNaN(value)) return console.error('The passed value to the scrollToPercents method should be number!');
+        if (value < 0) value = 0;
+        if (value > 100) value = 100;
```

Comment:
Don't we have the clamp function as a util?

Reply:
No but I added it as util to the scrollable container